README.md

```diff
@@ -62,29 +62,57 @@ Goad will read your credentials from `~/.aws/credentials` or from the `AWS_ACCES
 
 ```sh
 # Get help:
-$ goad -h
-Usage of goad:
-  -c uint
-      number of concurrent requests (default 10)
-  -m string
-      HTTP method (default "GET")
-  -n uint
-      number of total requests to make (default 1000)
-  -r string
-      AWS regions to run in (comma separated, no spaces) (default "us-east-1")
-  -t uint
-      request timeout in seconds (default 15)
-  -u string
-      URL to load test (required)
-  -H string
-      HTTP Header to add to the request. This option can be set multiple times.
+$ goad --help
+usage: goad --url=URL [<flags>]
+
+An AWS Lambda powered load testing tool
+
+Flags:
+  -h, --help                   Show context-sensitive help (also try --help-long
+                               and --help-man).
+  -u, --url=URL                URL to load test
+  -m, --method="GET"           HTTP method
+  -b, --body=BODY              HTTP request body
+  -c, --concurrency=10         Number of concurrent requests
+  -n, --requests=1000          Total number of requests to make
+  -N, --timelimit=3600         Seconds to max. to spend on benchmarking
+  -t, --timeout=15             Request timeout in seconds
+  -r, --region=REGION ...      AWS regions to run in (repeat flag to run in more
+                               then one region)
+  -p, --awsprofile=AWSPROFILE  AWS named profile to use
+  -o, --output=OUTPUT          Optional path to JSON file for result storage
+  -H, --header=HEADER ...      HTTP request header (repeat flag to add more then
+                               one header)
+  -s, --settings=.goad         Load settings from file (defaults to .goad in
+                               cwd)
+      --version                Show application version.
 
 # For example:
 $ goad -n 1000 -c 5 -u https://example.com
 ```
 
 Note that sites such as https://google.com that employ redirects cannot be tested correctly at this time.
 
+### Settings
+
+Goad supports to load settings stored in a ini compatible ([toml][]) settings file. By default if looks
+for a goad.ini file in the current working directory. Flags explicitly set on the commandline will
+be overwritten.
+
+```toml
+url = "http://example.com/"
+timeout = 15
+concurrency = 10
+requests = 100
+timelimit = 60
+method = "GET"
+body = "Hello world"
+regions = ["eu-west-1", "us-east-1"]
```

Comment:
not the format we discussed, please check https://github.com/zeph/goad/blob/master/desenmerda-te/default.ini